scripts/ansible/roles/linux/openenclave/vars/ubuntu/focal.yml

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+---
+gcc_target_version: "9.3.0"
+clang_target_version: "8.0.1"
```

Comment:
Can you double check the clang version?
When I do pull the latest ubuntu docker image (which is focal based), and then do `apt install clang`, the version it installs is `clang-10`. It would be better to install that version of clang as well for Ubuntu 20.04 image.

Reply:
I was installing this version so the CI images are consistent. Would you be okay hold off on this until all the CI images are upgraded to clang-10 or 11?